lib/v2/screens/app/interactor/viewmodels/app_bloc.dart

```diff
@@ -71,4 +57,30 @@ class AppBloc extends Bloc<AppEvent, AppState> {
     _shouldShowCancelGuardianAlertMessage.cancel();
     return super.close();
   }
+
+  void initDynamicLinks() async {
+    FirebaseDynamicLinks.instance.onLink(onError: (error) async {
+      print('onLinkError');
+      print(error.message);
+    }, onSuccess: (dynamicLink) async {
+      final Uri? deepLink = dynamicLink?.link;
+
+      if (deepLink != null) {
+        print('onSuccess');
+        print(deepLink.toString());
+        print(deepLink.data.toString());
```

Comment:
Same here.

Reply:
yeah, debug prints. Ill remove on the last PR. i need them for next pr